test/NuGet.Clients.Tests/NuGet.VisualStudio.Implementation.Test/Extensibility/VsSemanticVersionTests.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Xunit;
+
+namespace NuGet.VisualStudio.Implementation.Test.Extensibility
+{
+    public class VsSemanticVersionTests
+    {
+        [Fact]
+        public void SemanticVersion_ParseVersion_ThrowsIfNullOrEmpty()
+        {
+            var nullException = Assert.Throws<ArgumentException>(paramName: "version", () => SemanticVersion.Parse(null));
+            var emptyException = Assert.Throws<ArgumentException>(paramName: "version", () => SemanticVersion.Parse(string.Empty));
```

Comment:
I am good with this change but somehow, I think we can use `InlineData` attribute here with 2 values (null or empty string). I am fine either ways :)

Reply:
Good idea, done!